src/components/header/script.ts

```
@@ -0,0 +1,173 @@
+(() => {
+  "use strict";
+
+  try {
+    // Configuration
+    const ANIMATION_DURATION = 300;
+    const SCROLL_THRESHOLD = 50; // Pixel de scroll avant activation de l'effet
+
+    // Cache des éléments DOM
+    const elements = {
+      mobileMenu: document.getElementById("mobile-menu")!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)